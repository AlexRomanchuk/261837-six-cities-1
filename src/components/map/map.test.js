import React from "react";
import renderer from "react-test-renderer";
import Map from "../map/map.jsx";

const mockOffers = [
  {
    location: {
      coords: [52.3909553943508, 4.85309666406198]
    }
  }
];

const city = {
  coordinates: {
    coords: [52.3809553943508, 4.939309666406198]
  }
};

it(`correct renders map page`, () => {
  const tree = renderer
    .create(<Map
      places={mockOffers}
      city={city}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
