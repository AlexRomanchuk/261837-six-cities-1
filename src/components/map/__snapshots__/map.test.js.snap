// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correct renders map page 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  Не удалось показать карту
</div>
`;
